src/main/avro/boliglan.avsc

```diff
@@ -47,6 +47,13 @@
     {
       "name": "sikkerheteiendom",
       "type": ["null", "no.penger.export.domain.Boligdata"]
+    },
+    {
+      "name": "vedleggshasher",
```

Comment:
Can we reduce the name to "vedlegg", and in the "doc" field describe that this is a list of hashes?